packages/lit-dev-content/site/tutorials/content/custom-attribute-converter/03.md

```diff
@@ -0,0 +1,74 @@
+The current implementation can be greatly improved:
+
+- There is an unneccessary input of `dateStr` that has the job to accept a string
+- The code around `willUpdate` is a bit messy and imperative
+- If the user sets `date` to a new date, the `dateStr` will not be updated
+
+The root cause of these issues is that Lit does not have a default converter that converts a `string` to a `Date` object. This is where Lit attribute converters come in!
```

Comment:
Maybe add "custom" here? I would think just "Lit attribute converters" would include default ones which are not enough for us at the moment.
```suggestion
The root cause of these issues is that Lit does not have a default converter that converts a `string` to a `Date` object. This is where custom Lit attribute converters come in!
```